LEGO1/mxgeometry/mxgeometry3d.h

```diff
@@ -170,7 +170,7 @@ int UnknownMx4DPointFloat::BETA_1004aaa0(Matrix4& p_matrix, float p_f)
 }
 
 // FUNCTION: LEGO1 0x10004520
-inline long UnknownMx4DPointFloat::FUN_10004520()
+long UnknownMx4DPointFloat::FUN_10004520()
```

Comment:
This shouldn't make a difference functionally - the function will be treated as to be potentially inlined no matter what. For consistency we have been adding that keyword for header-defined functions where the body is outside the class definition (due to its size)

Reply:
yeah, you're right - I made a mistake when comparing and it looked like it made a change even when it didn't